packages/ng/forms/textfield/textfield.component.ts

```diff
@@ -47,6 +49,9 @@ export class TextfieldComponent extends AbstractFieldComponent {
 	@Input({ transform: booleanAttribute })
 	hasSearchIcon = false;
 
+	@Input({ transform: booleanAttribute })
+	hasTogglePasswordVisibilityIcon = false;
```

Comment:
Do we really need this input ? We could always use visibility icon on password input